Echo.json5

```diff
@@ -35,7 +35,7 @@
     { name: 'AREnableNewOnboardingFlow', value: true },
     { name: 'AREnableNewRequestPriceEstimateLogic', value: true },
     { name: 'AREnableNewSearch', value: true },
-    { name: 'AREnableNewTermsAndConditions', value: false },
+    { name: 'AREnableNewTermsAndConditions', value: true }, // 2024-09-03 removed artsy/eigen#10696
```

Comment:
👀 why are you flipping this to true if you are retiring it and it was false before?

Reply:
I assumed that was supposed to be released a long time ago and got lost maybe?

Reply:
@gkartalis so this feature was never actually released...? 😱

Reply:
I guess so since it is not flipped to true

Reply:
@gkartalis thanks for catching this. I've reached out to our PM to let them know that we failed to launch this feature back when, and will launch it with this PR instead.

Reply:
sounds good